content/sd/0003-tempo-e-sincronizacao.md

```diff
@@ -170,6 +180,36 @@ servidor de tempo falhar, o sistema não consegue sincronizar os relógios.
 Cristian propõe que o pedido seja feito em _multicast_ a vários servidores de
 tempo, selecionando o que responder primeiro, resultando na melhor precisão.
 
+:::info[Exercício]
+
+Um cliente pretende sincronizar-se com um servidor e para tal regista o RTT e os tempos
+enviados pelo servidor:
+
+| RTT (ms)            | Time (hh\:mm\:ss)             |
+| ------------------- | ----------------------------- |
+| <center>22</center> | <center>10:54:23.674</center> |
+| <center>25</center> | <center>10:54:25.450</center> |
+| <center>20</center> | <center>10:54:28.342</center> |
+
+> **1\. Com qual dos valores deve o servidor se sincronizar de modo a obter a melhor precisão?**
+
+**R:** Com o terceiro (10:54:28.342), pois apresenta o menor RTT ($20~ms$)
+
+> **2\. Qual o valor do relógio após o acerto?**
+
+**R:** $C(t) = t + \frac{RTT}{2} =$ 10:54:28.342 $+ \frac{20ms}{2} =$ 10:54:28.352
+
+> **3\. Qual a precisão do acerto?**
+
+**R:** $\pm~10~ms$
+
+> **4\. E se soubermos que o tempo de envio de uma mensagem é no mínimo de 8ms,
+> essa precisão é alterada? Se sim, qual o novo valor?**
+
+**R:** $\pm~(\frac{RTT-RTT_{min}}{2}) = \pm~(\frac{20}{2}-8) = \pm~2~ms$
```

Comment:
```suggestion
**R:** $\pm~(\frac{RTT-RTT_{min}}{2}) = \pm~(\frac{20-8}{2}) = \pm~6~ms$
```

Reply:
The result is actually correct now because $8ms$ is the time it takes to send the message, so $\frac{RTT}{2}$ instead of $RTT$. But it was wrong at the beginning when the formula was $\frac{RTT}{2}-RTT_{min}$ yes 😅